.github/workflows/os-runner.yml

```diff
@@ -0,0 +1,40 @@
+name: OS Runner Image
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - 'util/os-runner/**'
+
+env:
+  IMAGE_NAME: os-runner
+  IMAGE_TAG: latest
+  IMAGE_ID: ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        architecture:
+          - linux/amd64
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Build image
+        run: docker buildx build --platform ${{ matrix.architecture }} -t $IMAGE_NAME ./util/os-runner
+
+      - name: Log in to registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
+
+      - name: Push image
+        run: |
```

Comment:
Use the build and push [action](https://github.com/edi33416/docker-buildx-ci/blob/main/.github/workflows/ci.yml#L44) instead. This will remove the matrix definition and will take care of properly defining the docker manifest file

Reply:
Done